build_tools/travis/install.sh

```diff
@@ -86,18 +88,19 @@ elif [[ "$DISTRIB" == "pypy" ]]; then
     else
         ln -s "$BIN_PATH/pypy3" "$BIN_PATH/python"
     fi
+
     # add the binary to the path
     export PATH="$BIN_PATH:$PATH"
-    # install pip
-    python -m ensurepip
-    pip install -U pip wheel
+    wget https://bootstrap.pypa.io/get-pip.py
+    python get-pip.py    
+
     if [[ "$NUMPY" == "true" ]] && [[ "$PYTHON_VERSION" == "2.7" ]]; then
-        python -m pip install git+https://bitbucket.org/pypy/numpy.git
+        $BIN_PATH/python -m pip install git+https://bitbucket.org/pypy/numpy.git
     # numpypy does not work with pypy3 so fall back on numpy
     elif [[ "$NUMPY" == "true" ]]; then
-        python -m pip install cython numpy
+        $BIN_PATH/python -m pip install cython numpy
     fi
-    python -m pip install nose nose-timer pytest pytest-cov codecov python-dateutil
+    $BIN_PATH/python -m pip install nose nose-timer pytest pytest-cov codecov python-dateutil
 fi
 
 python --version
```

Comment:
Sure I can revert this back, I think the build is still broken anyway despite the green!